reconcile/statuspage/atlassian.py

```diff
@@ -64,16 +66,20 @@ def __init__(self, page_id: str, api_url: str, token: str):
             api_key=self.token, page_id=self.page_id, organization_id="unset"
         )
 
+    @retry(max_attempts=10)
+    def _do_get(self, url: str, params: dict[str, Any]) -> Response:
+        response = requests.get(url, params=params, headers=self.auth_headers)
```

Comment:
requests.get does not have a timeout per default, please specify one

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4337/commits/71b53096859b0be2ee82452f9c90446e6139d1e3